spec/requests/cloud_object_store_containers_spec.rb

```diff
@@ -45,4 +47,47 @@
       expect(response.parsed_body).to include(expected)
     end
   end
+
+  context 'POST /api/cloud_object_store_containers/' do
+    it 'it can create cloud object store containers through POST' do
+      zone = FactoryBot.create(:zone)
+      provider = FactoryBot.create(:ems_storage, :zone => zone)
+      cloud_tenant = FactoryBot.create(:cloud_tenant, :ext_management_system => provider)
+
+      api_basic_authorize collection_action_identifier(:cloud_object_store_containers, :create, :post)
+      submit_data = {
+        :ems_id          => provider.id,
+        :name            => 'foo',
+        :emsType         => "ManageIQ::Providers::Openstack::StorageManager::SwiftManager",
```

Comment:
Should be `provider.type` rather than hard-coding this string

Reply:
> Should be `provider.type` rather than hard-coding this string

Fixed this now, everything should be good just waiting for tests to go green